inception-project/inception
# #4631 - Enable pre-trained external recommenders that can also be trained on

**What's in the PR**
- Added ability to configure external recommender as "training supported"

**How to test manually**
* Configure training supported for an external recommender
* Check if prediction requests can be sent even if there was no training yet

**Automatic testing**

**Documentation**
